lib/safe-pg-migrations/base.rb

```diff
@@ -98,8 +101,11 @@ def exec_migration(connection, direction)
     end
 
     def disable_ddl_transaction
-      UselessStatementsLogger.warn_useless '`disable_ddl_transaction`' if super
-      true
```

Comment:
Small bugfix here: this log was never sent because current_migration is not set yet